plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMetadataFileOperations.java

```diff
@@ -384,10 +385,41 @@ public void testPredicateWithVarcharCastToDate()
         assertUpdate("DROP TABLE test_varchar_as_date_predicate");
     }
 
+    @Test
+    public void testRemoveOrphanFiles()
```

Comment:
are you planning on adding a test for snapshot expiration as well?

Reply:
I can, probably in a separate PR

Reply:
I guess this might be covered by @homar 's https://github.com/trinodb/trino/pull/14434